src/core/Akka.Remote.Tests/RemoteConfigSpec.cs

```diff
@@ -103,7 +103,7 @@ public void Remoting_should_contain_correct_heliosTCP_values_in_ReferenceConf()
         [Fact]
         public void When_remoting_works_in_Mono_ip_enforcement_should_be_defaulted_to_true()
         {
-            HeliosTransportSettings.IsMono = true;
+            if (!IsMono) return; // skip IF NOT using Mono
```

Comment:
Mono 4.4.2 can't handle dual-mode sockets correctly, so we have to force the socket to use IPV4 OR IPV6, not "either." We check inside our `RemoteSettings` class to determine that this is explicitly set.